assets/pages/new-contributions-landing/components/ContributionUsTicker.jsx

```diff
@@ -16,11 +16,11 @@ type PropTypes = {}
 // ---- Helpers ----- //
 
 const getInitialTickerValues = (): Promise<StateTypes> =>
-  fetch('https://interactive.guim.co.uk/docsdata-test/1ySn7Ol2NQLvvSw_eAnVrPuuRnaGOxUmaUs6svtu_irU.json')
+  fetch('https://support.theguardian.com/ticker.json')
```

Comment:
can you not just do `/ticker.json`? then it will work in CODE & dev

Reply:
and maybe it won't need the addition to the `contentSecurityPolicy`

Reply:
very good point!